src/main/haskell/language-kore/src/Data/Kore/ASTVerifier/SentenceVerifier.hs

```diff
@@ -162,6 +163,11 @@ verifyMetaSentence
     (SentenceAxiomSentence axiomSentence)
   =
     verifyAxiomSentence axiomSentence indexedModule attributesVerification
+verifyMetaSentence
+    _indexedModule
+    _attributesVerification
+    (SentenceSortSentence _sortSentence)
+  = error "Meta sort declaration well-formedness not yet defined"
```

Comment:
When do you intend to fix this?

Reply:
Should been in the next commit, was waiting for feedback, but I just did it